server/src/weaverbird/backends/mongo_translator/mongo_pipeline_translator.py

```diff
@@ -0,0 +1,25 @@
+from weaverbird.backends.mongo_translator.steps import mongo_step_translator
+from weaverbird.pipeline import Pipeline, PipelineStep
+
+
+def translate_pipeline(
+    pipeline_to_translate: Pipeline,
+) -> list:
+    mongo_pipeline = []
+    for index, step in enumerate(pipeline_to_translate.steps):
+        try:
+            mongo_pipeline.append(mongo_step_translator[step.name](step))
```

Comment:
Does append works like concat when we have lists ?
(To avoid having a list of lists in the end, which needs flattening)